### PR TITLE
Docs - What is EOP

### PR DESCRIPTION
**Pull request by smozely**

### CODE REVIEW

File: docs-site/src/markdown-pages/index.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+from multiple systems, which is slow, error-prone, and often this is done in a
+way that isn't consistent within a single council or across councils. This
+effort to integrate the data limits the type of analysis that people will even
+try to achieve with the data. This leads to the value of the data not being
+fully realised.
+
+Much work has been done in councils by people without a software or data
+engineering background on the best efforts' basis. This has resulted in
+solutions that deliver lots of value but lack the maintainability, scalability
+and user-centric approach that would come from a software engineering team.
```
Comment (smozely) [inline docs-site/src/markdown-pages/index.md:25]: Trying to say there are a lot of "Not great quality" solutions out there ... but not sure its needed or worded well.

Really using it as a line to introduce why bringing in a team that follows "Modern Development Practices" is an important step

File: docs-site/src/markdown-pages/index.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+- To provide guidance on issues or risks that are escalated from the EOP
+  Delivery Team Members.
+- To resolve any dispute or disagreement that is raised through the governance
+  structure.
+
+### Technical Working Group
+
+A technical group is going to be setup in 2023 led by the EOP Delivery team.
+This will be the forum to share and collaborate on the technical solution across
+contributing councils and potential future contributors.
```
Comment (smozely) [inline docs-site/src/markdown-pages/index.md:90]: Trying to word this in a way to make it clear that its "Governance by the people building the solution (or contributing data) / sharing and collaboration with those who are interested."

File: docs-site/src/markdown-pages/index.md (hunk 1 above)
Comment (vimto) [reply docs-site/src/markdown-pages/index.md:25]: Sounding good. And good idea to frame-up the value of "Modern Development Practices" in the last para.

I've taken a stab at word-smithing, but I'm not precious about it so feel free to use or ignore. It's mainly just breaking down long sentences and removing some duplication. I also bought back the statement explaining what it is (an end-to-end platform for capture...) in the opening sentence as that seemed like a good thing to start with?

> EOP is an initiative led by Greater Wellington Regional Council to establish an end-to-end platform for capture, management and reporting of environmental information. The goal is to allow all regional councils to maximise the value they get from their environmental monitoring efforts, and provide clear, consistent and user-centric information to end-users if this data. It’s built based on the experiences from the Environment Canterbury Water Data programme 
> 
> Regional councils have many systems for collecting and managing environmental data. These systems are often siloed and based on legacy technology. Integrating this data can be a slow, manual and and error prone process that is not consistent within or across councils. These challenges to limits the type of analysis that is feasible, meaning the value of the data is not fully realised.
> 
> Much work has been done in councils by people without a software or data engineering background on a best efforts' and as-needed basis. This has resulted in solutions that deliver lots of value but lack the maintainability, scalability and user-centric approach that would come from a dedicated software engineering team.

File: docs-site/src/markdown-pages/index.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+enable the effort to deliver an outcome by one council to be delivered to all
+councils for an incremental cost. This means in general, EOP will be agnostic of
+council internal systems with each council providing data to EOP in a
+standardised format. However, for some common systems used in a majority of
+councils, EOP can provide standardised adapters to allow the data to be
+
+_Modern Development Practices_
+
+We are using modern deliver practices ... why do you care? It supports long term
+maintainability and reduces the risk of the platform becoming legacy.
```
Comment (vimto) [inline docs-site/src/markdown-pages/index.md:48]: There's a switch to a different perspective here ("We"). Also, is it worth a very brief sentence explaining what some of these are? E.g.

> The EOP platform will be developed using modern development and delivery practices to reduce the risk of the platform becoming legacy, and help ensure future development and operation and is sustainable. This includes test coverage, infrastructure automation and a _devops_ approach, and selecting fit-for-purpose tools, infrastructure and standards.

File: docs-site/src/markdown-pages/index.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+### Technical Working Group
+
+A technical group is going to be setup in 2023 led by the EOP Delivery team.
+This will be the forum to share and collaborate on the technical solution across
+contributing councils and potential future contributors.
+
+- Setting the technical architecture for EOP
+- Setting the technical development practices for EOP
+- Setting the technical collaboration practices for EOP
+- Setting technical priorities for the EOP Delivery Team
```
Comment (vimto) [inline docs-site/src/markdown-pages/index.md:95]: Is "Setting technical priorities for the EOP Delivery Team" too hands on for this group? My take is this should be decided by the team, with input from this group, alongside the Steerco and PO/Roadmap. Also, a little word-smithing of the bullet points:

>We imagine the activities of this group to be:
>- Discussion and feedback on major architectural concerns and decisions
>- Helping establish development practices for technical collaboration on EOP across councils
>- Feeding back on current technical priorities of the EOP Delivery Team
>- Bringing a cross-council perspective to technical development by sharing knowledge, challenges and opportunities from different councils

**Review by vimto — CHANGES_REQUESTED** [review]
Great work on the revamp, it's sounding good. I've added some comments and suggestion, mostly just some tweaking words without really changing the bones of it.

**Review by vimto — APPROVED** [review]

**Pull request merged**